pages/referencing-the-text.xml

```diff
@@ -0,0 +1,169 @@
+                <p/>
+            </publicationStmt>
+            <sourceDesc>
+                <p>Produced by pandoc. Conversion from GIT flavoured markdown to TEI simple.</p>
+                <p>transformed to TEI documentation with XSLT by Pietro Liuzzo</p>
+            </sourceDesc>
+        </fileDesc>
+        <revisionDesc>
+            <change who="PL" when="2018-04-30">first version of guidelines from Wiki</change>
+            <change who="PL" when="2020-06-16">introduced text and example for referencing based on
+                DTS navigation and document support</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body xml:id="referencing-the-text">
+
+            <div type="level2">
+                <head>Canonical text citations and quotations in text</head>
+                <p>For citations like Matthew 14, 3, use <gi>ref</gi> with a <att>cRef</att>
+                    containing the URN which you can get from the Perseus catalogue searching for
+                    the passage. This will link to the Greek text and should therefore only be used
+                    when it is actually the Greek text that is intended.</p>
+                <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                    <ref cRef="urn:cts:greekLit:tlg0031.tlg001.perseus-grc1:14.3">Matthew 14,
+                        3</ref>
+                </egXML>
+                <p>You can also point to our texts, for example the Ethiopic text of Numbers 21, 8
+                    like this:</p>
+                <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                    <ref cRef="betmas:LIT2075Number.21.8">Numbers 21:8</ref>
+                </egXML>
+                <p>To mark up a passage in the text which is a literal quotation of another text you
+                    might use a <gi>cit</gi> containing a <gi>quote</gi> and the above <gi>ref</gi>
+                </p>
+                <p>please note the difference! In this case we say that the passage marked is a
+                    quotation of another text, whilest ref is used here and there only to encode the
+                    actual canonical reference.</p>
+                <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                    <cit>
+                        <quote xml:lang="gez"> ለሙሴ᎓ ግበር᎓ ለከ᎓ አርዌ᎓ ምድር᎓ ዘብረት᎓ ኵሉ᎓ ዘነሰኮ᎓ ይርአይ᎓ ኪያሁ᎓
+                            ወይሕየው᎓ ኪዮስ᎓ </quote>
+                        <ref cRef="betmas:LIT2075Number.21.8">Numbers 21:8</ref>
+                    </cit>
+                </egXML>
+                <p>The following example of markup is confusing and should be avoided as it claims
+                    that the content of the ref element is a reference, while it is a quote.</p>
+                <egXML xmlns="http://www.tei-c.org/ns/Examples"> ለሙሴ᎓ <ref
+                        cRef="betmas:LIT2075Number.21.8">ግበር᎓ ለከ᎓ አርዌ᎓ ምድር᎓ ዘብረት᎓ ኵሉ᎓ ዘነሰኮ᎓ ይርአይ᎓
+                        ኪያሁ᎓ ወይሕየው</ref>᎓ ኪዮስ </egXML>
```

Comment:
Maybe we could delete this?

Reply:
as above, I think it does not harm and may help prevent an understandable misunderstanding in the future